packages/@aws-cdk-testing/framework-integ/test/aws-batch/test/integ.ecs-job-definition.js.snapshot/tree.json

```diff
@@ -1616,7 +1620,7 @@
               }
             },
             "constructInfo": {
-              "fqn": "@aws-cdk/aws-batch-alpha.EcsJobDefinition",
+              "fqn": "aws-cdk-lib.aws_batch.EcsJobDefinition",
```

Comment:
Do we know why this changed?

Reply:
The last change of this file was to stabilizing (convert alpha to regular module). So the snapshot would be before stabilizing which was still using `aws-batch-alpha`